### PR TITLE
Update test result file default in Gradle task

### PR DESCRIPTION
**Pull request by madhurig**

### CODE REVIEW

File: Tasks/Gradle/task.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 134,
+        "Minor": 135,
```
Comment (madhurig) [inline Tasks/Gradle/task.json]: Could be considered a behavior change, but not updating major version. David pointed out the side effect would be more test results are uploaded than expected and it is easy to update to be more specific.

**Comment by DavidStaheli** [comment]
@madhurig thank you!

File: Tasks/Gradle/task.json (hunk 1 above)
Comment (lkillgore) [reply Tasks/Gradle/task.json]: Only for *new* definitions, correct?  Either way, I agree that this doesn't need a major version bump.

Comment (madhurig) [reply Tasks/Gradle/task.json]: I verified that the default does not change for existing definitions/tasks since it is a string value.

**Pull request merged**